build/scripts/testing.mk

```diff
@@ -202,7 +202,7 @@ start-erigon: ## start an ephemeral `erigon` node
 	docker run \
     --rm -v $(PWD)/${TESTAPP_FILES_DIR}:/${TESTAPP_FILES_DIR} \
     -v $(PWD)/.tmp:/.tmp \
-    thorax/erigon:latest init \
+    erigontech/erigon:latest init \
```

Comment:
merged https://github.com/berachain/beacon-kit/pull/2121 to unlock CI compilation